news/2024/2024-01-16-mappers-guild-more-priority-quests.md

```diff
@@ -0,0 +1,104 @@
+
+### [The Hurdle v2](https://mappersguild.com/missions?id=65a6e6a3fa448adc05ab6bc5)
+
+As any new mapper can attest, this might be the hardest quest of all.
+
+Create and rank your first beatmap.
+
+Good luck.
+
+## Other changes to the Mappers' Guild
+
+The **Mappers' Guild** is an ever-changing system. In addition to this big batch of quests, we have one major change to publicise:
+
+**Hitsounds can be separately added to Mappers' Guild beatmaps!**
+
+![](/wiki/shared/news/2024-01-16-mappers-guild-more-priority-quests/hitsounds.png)
+
+In the past, a map's host almost always created hitsounds, but trends have changed lately. Nowadays, more and more mappers ask other mappers to create hitsounds for their maps, sort of like [guest difficulties](/wiki/Beatmap/Guest_difficulty). 
+
+The Mappers' Guild is following this trend, allowing guest hitsounders to earn the rewards they deserve!
+
+---
+
+That's all for now! If you're still reading, you should really be [checking out the new priority quests](https://mappersguild.com/missions) instead.
+
+And of course, congrats to these folks for winning one of the previous priority quests: ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976), ::{ flag=ID }:: [Ucitysm](https://osu.ppy.sh/users/14768693), ::{ flag=AU }:: [fvrex](https://osu.ppy.sh/users/11863699), ::{ flag=IE }:: [HEAVENLY MOON](https://osu.ppy.sh/users/13681283), ::{ flag=US }:: [Nao Tomori](https://osu.ppy.sh/users/5364763), ::{ flag=IT }:: [zekk](https://osu.ppy.sh/users/9704802), ::{ flag=ID }:: [Ilham](https://osu.ppy.sh/users/3057154), ::{ flag=HK }:: [Cynplytholowazy](https://osu.ppy.sh/users/3901754), ::{ flag=UZ }:: [M a r v o l l o](https://osu.ppy.sh/users/5504231), ::{ flag=DE }:: [Stardust Prism](https://osu.ppy.sh/users/8525921), ::{ flag=HU }:: [Nytrocide_](https://osu.ppy.sh/users/11327918), ::{ flag=FI }:: [Antti](https://osu.ppy.sh/users/13281473), ::{ flag=US }:: [fieryrage](https://osu.ppy.sh/users/3533958), ::{ flag=BR }:: [AutotelicBrown](https://osu.ppy.sh/users/4238941), ::{ flag=ID }:: [Hinsvar](https://osu.ppy.sh/users/1249323), ::{ flag=PL }:: [Zelq](https://osu.ppy.sh/users/8953955), ::{ flag=DE }:: [Lasse](https://osu.ppy.sh/users/896613), ::{ flag=RU }:: [Daycore](https://osu.ppy.sh/users/5596337), ::{ flag=BE }:: [Sylas](https://osu.ppy.sh/users/3906405), ::{ flag=ES }:: [Deif](https://osu.ppy.sh/users/318565), ::{ flag=CA }:: [wwwww](https://osu.ppy.sh/users/8434466), ::{ flag=NO }:: [Chrisse](https://osu.ppy.sh/users/18382591), ::{ flag=CA }:: [Unlucky_w](https://osu.ppy.sh/users/4820793), ::{ flag=SG }:: [_gt](https://osu.ppy.sh/users/8301957), ::{ flag=PH }:: [Jemzuu](https://osu.ppy.sh/users/7890134), ::{ flag=CN }:: [iBell](https://osu.ppy.sh/users/4298072), ::{ flag=NL }:: [Saemitsu](https://osu.ppy.sh/users/14262789), ::{ flag=HK }:: [DraterTD](https://osu.ppy.sh/users/11671725), ::{ flag=FR }:: [Ventilo le vrai](https://osu.ppy.sh/users/13661408)
```

Comment:
`::{ flag=UZ }:: [M a r v o l l o](https://osu.ppy.sh/users/5504231)` link points to Shmiklak

Reply:
name change